src/warzoneconfig.h

```diff
@@ -67,8 +67,8 @@ void war_SetFMVmode(FMV_MODE mode);
 FMV_MODE war_GetFMVmode();
 void war_SetAllowSubtitles(bool);
 bool war_GetAllowSubtitles();
-void war_setFullscreen(bool);
-bool war_getFullscreen();
+void war_setFullscreen(WINDOW_MODE);
```

Comment:
What if we rename these functions to `war_getWindowMode` and `war_setWindowMode`?